### PR TITLE
Release note for Cache Api use in webextensions

### LINKED ISSUE #5378: [WebExt] Now able to use Cache API in WebExtensions
## Acceptance criteria

1. - [ ] The listed features are documented sufficiently on MDN
2. - [ ] BCD is updated
3. - [ ] Interactive example and data repos are updated if appropriate
4. - [ ] The content has been reviewed as needed

## Features to document

1. Now able to use Cache API in WebExtensions

## Related Gecko bugs
1. - [ ] [moz-extension:// schema is not considered secure for Cache Web API](https://bugzilla.mozilla.org/show_bug.cgi?id=1575625). Looks like a rel note is definitely needed to say that Cache API can now be used in WebExt, and what's changed. Not 100% sure if this is worth mentioned in Cache API docs, but it should probably be mentioned somewhere appropriate in the WebExt docs.

For folks helping with Firefox-related documentation features — make sure above AC have been done, but also:

* Set bugs to dev-doc-complete
* Add entry to Firefox release notes if feature is enabled in release
* Add entry to Firefox experimental features page if feature is not yet enabled in release

### PR DESCRIPTION
**Pull request by mirunacurtean**
> What was wrong/why is this fix needed? (quick summary only)

Fixes https://github.com/mdn/content/issues/5378

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/90/index.html [hunk 1]
```diff
@@ -63,6 +63,10 @@ <h4 id="removals_sec">Removals</h4>
 
 <h3 id="APIs">APIs</h3>
 
+<ul>
+  <li>Starting with this version, the <a href= "/en-US/docs/Web/API/Cache">Cache API</a> can be used in the extension page (for content script injection) and the worker globals. For more details see ({{bug(1575625)}}).</li>
```
Comment (rpl) [inline files/en-us/mozilla/firefox/releases/90/index.html:67]: Remove `(for content script injection)`, the Cache API available in a content script is the one from the attached webpage (and the changes from Bug 1575625 are unrelated to that, the patch landed did actually enable using it from extension pages, like the background page or a browserAction popup, moz-extension workers and to allow caching moz-extension urls, (this part did not work before that bug was fixed).

**Review by rpl — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/firefox/releases/90/index.html (hunk 1 above)
Comment (mirunacurtean) [reply files/en-us/mozilla/firefox/releases/90/index.html]: @rpl Thanks for the clarifications. I remove what you requested and added further details that you provided on the scope of this fix. Hope it looks good now!

File: files/en-us/mozilla/firefox/releases/90/index.html [hunk 2]
```diff
@@ -63,6 +63,10 @@ <h4 id="removals_sec">Removals</h4>
 
 <h3 id="APIs">APIs</h3>
 
+<ul>
+  <li>Starting with this version, the <a href= "/en-US/docs/Web/API/Cache">Cache API</a> can be used in the extension pages (like background page, browserAction popups, moz-extension workers or allowing the caching of moz-extension urls) and the worker globals. For more details, see ({{bug(1575625)}}).</li>
```
Comment (rpl) [inline files/en-us/mozilla/firefox/releases/90/index.html:67]: Nit, personally I think we may omit the details from the release notes and leave it as:
```
Starting with this version, the Cache API can be used in the extension pages and worker globals. For more details, see ... 
```

**Review by rpl — APPROVED** [review]
looks good, but as described below I think we may omit the additional details from the release notes and keep the description in this list item simple as it was in the previous version (just without the reference to the content script).

(I mainly described why the note on the content script was wrong for completeness of the review comment).

File: files/en-us/mozilla/firefox/releases/90/index.html (hunk 2 above)
Comment (Rumyra) [reply files/en-us/mozilla/firefox/releases/90/index.html:67]: I'm going to merge this as I still have to do an edit of the release notes this afternoon - @rpl I'll modify for what you've said in that pr 👍

**Review by Rumyra — APPROVED** [review]

**Pull request merged**